packages/flutter/lib/src/material/expansion_panel.dart

```diff
@@ -111,9 +120,82 @@ class ExpansionPanelRadio extends ExpansionPanel {
 /// A material expansion panel list that lays out its children and animates
 /// expansions.
 ///
+/// {@tool snippet template=stateful_widget_material}
```

Comment:
use `--template`. It works without the `--`, but that's kind of just for legacy reasons.